backend/expensia/expense/models.py

```diff
@@ -11,3 +14,30 @@ def __str__(self):
 
     def get_absolute_url(self):
         return reverse('expense:category-detail', args=(self.pk,))
+
+
+class Expense(models.Model):
+    """ Expense model """
+
+    name =models.CharField(_("Name"), max_length=30, blank=False, null=False)
+    start_date = models.DateTimeField(_("Start Date"), null=False, blank=False)
```

Comment:
Expense just has a creation date no need for start and end date here.
Just inherit from `TimeStampedModel`

Don't forget to remove these two fields in other places.
Also delete the migration and create another one